packages/proc-smaps/proc-smaps.0.2.0/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+synopsis: "Proc-smaps: An ocaml parser of /proc/[pid]/smaps"
+maintainer: "Hiromasa Saito <hiromasa.saito@dailambda.jp>"
+authors: "Hiromasa Saito <hiromasa.saito@dailambda.jp>"
+license: "MIT"
+homepage: "https://gitlab.com/dailambda/proc-smaps/"
+bug-reports: "https://gitlab.com/dailambda/proc-smaps/-/issues"
+depends: [
+  "ocaml" {>= "4.8.0"}
+  "dune" {>= "2.9.0"}
+  "lwt" {>= "5.4.0"}
+  "stdint"
+  "alcotest" {with-test}
+]
```

Comment:
```suggestion
]
conflicts: [
  "result" {< "1.5"}
]
```
otherwise it fails with:
```
#=== ERROR while compiling proc-smaps.0.2.0 ===================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.4.08.1 | pinned(https://gitlab.com/dailambda/proc-smaps/-/archive/0.2.0/proc-smaps-0.2.0.tar.bz2)
# path                 ~/.opam/4.08/.opam-switch/build/proc-smaps.0.2.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p proc-smaps -j 255
# exit-code            1
# env-file             ~/.opam/log/proc-smaps-7-1adbf3.env
# output-file          ~/.opam/log/proc-smaps-7-1adbf3.out
### output ###
#       ocamlc src/.smaps.objs/byte/smaps.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.smaps.objs/byte -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/lwt -I /home/opam/.opam/4.08/lib/lwt/unix -I /home/opam/.opam/4.08/lib/mmap -I /home/opam/.opam/4.08/lib/ocaml/threads -I /home/opam/.opam/4.08/lib/ocplib-endian -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/stdint -intf-suffix .ml -no-alias-deps -o src/.smaps.objs/byte/smaps.cmo -c -impl src/smaps.ml)
# File "src/smaps.ml", line 155, characters 17-28:
# 155 |     let (let*) = Result.bind in
#                        ^^^^^^^^^^^
# Error: Unbound value Result.bind
#     ocamlopt src/.smaps.objs/native/smaps.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlopt.opt -w -40 -g -I src/.smaps.objs/byte -I src/.smaps.objs/native -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/lwt -I /home/opam/.opam/4.08/lib/lwt/unix -I /home/opam/.opam/4.08/lib/mmap -I /home/opam/.opam/4.08/lib/ocaml/threads -I /home/opam/.opam/4.08/lib/ocplib-endian -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/stdint -intf-suffix .ml -no-alias-deps -o src/.smaps.objs/native/smaps.cmx -c -impl src/smaps.ml)
# File "src/smaps.ml", line 155, characters 17-28:
# 155 |     let (let*) = Result.bind in
#                        ^^^^^^^^^^^
# Error: Unbound value Result.bind
```

Reply:
Thank you for your suggestion :pray: